src/line_opacity.jl

```diff
@@ -99,7 +151,8 @@ end
 function voigt(α, v)
     if α <= 0.2 
         if (v >= 5)
-            (α / sqrt(π) / v^2) * (1 + 3/(2v^2) + 15/(4v^4))
+            invv2 = (1/v)^2
+            (α/sqrt(π) * invv2) * (1 + 1.5invv2 + (15/4)*invv2^2)
```

Comment:
I strongly doubt that.

As a general rule of thumb, AOT compilers can't evaluate floating point operations at compile time (even if they involve integers). I strongly doubt that a JIT would spend extra time handling this sort of thing. 

It _might_ be able to handle 15 ÷ 4 (integer divsion, i.e. the answer would be 3). But even then, I'm not so sure since division is a complicated operation (I'm only confident that a compiler can perform addition, subtraction, and multiplication of integer constants at compile time).

Reply:
I just verified that you are correct with `@code_lowered`.  This is surprising to me!

Reply:
Actually, per `@code_llvm` these should produce the same result.  Not that it matters.  
```    
f(x) = (15/4) * x
g(x) = 3.75 * x

@code_llvm f(2)
;  @ In[31]:1 within `f'
define double @julia_f_7412(i64 signext %0) {
top:
; ┌ @ promotion.jl:322 within `*'
; │┌ @ promotion.jl:292 within `promote'
; ││┌ @ promotion.jl:269 within `_promote'
; │││┌ @ number.jl:7 within `convert'
; ││││┌ @ float.jl:94 within `Float64'
       %1 = sitofp i64 %0 to double
; │└└└└
; │ @ promotion.jl:322 within `*' @ float.jl:332
   %2 = fmul double %1, 3.750000e+00
; └
  ret double %2
}
lvm

@code_llvm g(2)
;  @ In[31]:2 within `g'
define double @julia_g_7440(i64 signext %0) {
top:
; ┌ @ promotion.jl:322 within `*'
; │┌ @ promotion.jl:292 within `promote'
; ││┌ @ promotion.jl:269 within `_promote'
; │││┌ @ number.jl:7 within `convert'
; ││││┌ @ float.jl:94 within `Float64'
       %1 = sitofp i64 %0 to double
; │└└└└
; │ @ promotion.jl:322 within `*' @ float.jl:332
   %2 = fmul double %1, 3.750000e+00
; └
  ret double %2
}
```